src/change_stream.ts

```diff
@@ -600,10 +616,10 @@ function endStream(changeStream: ChangeStream): void {
   changeStream[kCursorStream] = undefined;
 }
 
-function processNewChange(
-  changeStream: ChangeStream,
-  change: ChangeStreamDocument,
-  callback?: Callback
+function processNewChange<TSchema>(
+  changeStream: ChangeStream<TSchema>,
+  change: Nullable<ChangeStreamDocument<TSchema>>,
```

Comment:
This is the only place I used `Nullable` a custom type defined in `mongo_types.ts` to mirror the global typescript `NonNullable`. It feels like something that has come up in a few places, the alternative would be `change: ChangeStreamDocument<TSchema> | undefined | null`.